plugins/woocommerce-admin/client/marketplace/components/tabs/tabs.tsx

```diff
@@ -110,26 +132,31 @@ const renderTabs = ( contextValue: MarketplaceContextType ) => {
 const Tabs = ( props: TabsProps ): JSX.Element => {
 	const { additionalClassNames } = props;
 	const marketplaceContextValue = useContext( MarketplaceContext );
-	const { setSelectedTab } = marketplaceContextValue;
+	const { selectedTab, setSelectedTab } = marketplaceContextValue;
+	const [ activeTabs, setActiveTabs ] = useState( getActiveTabs( '' ) );
 
 	const query: Record< string, string > = useQuery();
+	const queryLoaded = Object.keys( query ).length > 0;
```

Comment:
I feel like `queryLoaded` should be defined in the `useEffect`. Every time `query` changes, `useEffect` will run and will re-calculate the `queryLoaded`. Although didn't notice any issues on the page with the current configuration.

Reply:
Done, I removed the var.
I was thinking that this would make it more clear what `Object.keys( query ).length > 0` means, but it's not needed.